src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -499,4 +516,57 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
     private Class loadBindClass() {
         loadClass("${MODEL_PACKAGE}.Bind")
     }
+    
+    private def parseDevice(String deviceString) {
+        List<String> tokens = deviceString.tokenize(':')
+        int numberOfTokens = tokens.size()
```

Comment:
Do we need to create this extra variable? Can we not just use the size in the switch itself:

    switch(tokens.size()) {
        ...
    }